test/modules/load-gen/PriorityMessageSender.cs

```diff
@@ -52,8 +61,10 @@ public async override Task RunAsync(CancellationTokenSource cts, DateTime testSt
                 {
                     int choosePri = this.rng.Next(outputs.Length);
```

Comment:
Either you check in the constructor to ensure that both priorities and ttls has same number of items; or use minimum size of priorities and ttls, e.g. Math.Min(priorities.Length, ttls.Length).  If you choose to ensure both have same number of arguments and with previous comment suggest to convert these to collection type in settings class; you should do this validation in Settings constructor.

Reply:
or you can think of another structure, for example 1 environment variable "MessagePriorityAndTTL" has value of <priority1>,<ttl1>;<priority2>,<ttl2>...

Reply:
Refactored using TestInfo struct